.circleci/test-deploy.yml

```diff
@@ -6,18 +6,30 @@ orbs:
 filters: &filters
   tags:
     only: /.*/
-
-jobs:
-  command-tests:
+executors:
+  alpine:
+    docker: 
+      - image: alpine:latest
+  docker-base:
     docker:
-      - image: cimg/base:current
-    steps:
-      - aws-health/check-health
+      - image: cimg/base:stable
+  windows:
+    machine:
+      image: windows-server-2022-gui:current
+    shell: bash.exe
```

Comment:
i would probably check with mike on if bash vs powershell matters here? i'm not sure